packages/rum/src/domain/record/elementsScrollPositions.ts

```diff
@@ -1,20 +1,29 @@
 import { addTelemetryDebug } from '@datadog/browser-core'
+import { forEach } from './utils'
 
 export type ElementsScrollPositions = ReturnType<typeof createElementsScrollPositions>
 export type ScrollPositions = { scrollLeft: number; scrollTop: number }
 
 export function createElementsScrollPositions() {
   const scrollPositionsByElement = new WeakMap<Element, ScrollPositions>()
-  const documentScrollingElement = document.scrollingElement!
-  if (!documentScrollingElement) {
-    addTelemetryDebug('document without scrollingElement')
-  }
+  let documentScrollingElement = document.scrollingElement
```

Comment:
From MDN documentation, “In standards mode, [`document.scrollingElement`] is the root element of the document, [document.documentElement](https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement).”. 

Thus, in standard mode, we can fallback to `document.documentElement` if we don't have `scrollingElement`.

<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ how far should we go to support non-standard modes? Is it even supported by the player?